bench/bench_eb.c

```diff
@@ -153,13 +153,13 @@ static void util(void) {
 }
 
 static void arith(void) {
-	eb_t p, q, r, t[EB_TABLE_MAX];
+	eb_t p, q, r, t[EB_RELIC_TABLE_MAX];
```

Comment:
I wanted to have RELIC_ with the context of TABLE, but if it's better as a proper prefix I can change it again?